libsql-client/src/cell_value.rs

```diff
@@ -0,0 +1,57 @@
+//! `CellValue` represents libSQL values and types.
+//! Each database row consists of one or more cell values.
+
+/// Value of a single database cell
+#[derive(Clone, Debug)]
+pub enum CellValue {
+    Text(String),
+    Float(f64),
+    Number(i64),
+    Bool(bool),
+    Null,
+}
+
+impl std::fmt::Display for CellValue {
+    fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
+        match self {
+            CellValue::Text(s) => write!(f, "\"{s}\""),
```

Comment:
I think that we should either use the `Display` impl of `String` (`"{}"`), or the `Debug` impl (`"{:?}"`). This "poor man's formatting" will produce odd results for strings that contain characters like `"` or `\`.

Reply:
In the longer term I think it should be simply unified with the server part, i.e. https://github.com/libsql/sqld/blob/main/sqld/src/http/types.rs . This code is a terrible oversimplification and abuse of std::fmt::Display, but it's a start :innocent: